### PR TITLE
build: Form filters (Do Not Merge)

### PR DESCRIPTION
**Pull request by istarkov**
# Example
Edit:
https://form-filters.staging.webstudio.is/builder/4598a9d6-dfcb-4445-a2a4-f4f790d0a329
Published:
https://saas-filters-gbb8i.wstd.work/

See steps below
....

### CODE REVIEW

File: apps/builder/app/builder/features/settings-panel/resource-panel.tsx [hunk 1]
```diff
@@ -296,7 +296,7 @@ const $selectedInstanceScope = computed(
     if (values) {
       for (const [dataSourceId, value] of values) {
         if (hiddenDataSourceIds.has(dataSourceId)) {
-          continue;
+          // continue;
```
Comment (TrySound) [inline apps/builder/app/builder/features/settings-panel/resource-panel.tsx:299]: rebase and uncomment

**Comment by istarkov** [comment]
# Hygraph

Login to your hygraph
https://app.hygraph.com/

At dashborad create new project using  Commerce E-Shop template
image

Using API Playground we will write queries used in project above
image


```graphql
query Products($where: ProductWhereInput, $currentCategoryWhere:CategoryWhereInput, $locales: [Locale!]!) {
  
  currentCategory: categories(where: $currentCategoryWhere, locales: $locales) {
slug    localizations(includeCurrent: true, locales: [en, de]) {
    	slug
    }
  }
  
    categories(locales: $locales) {
      slug
      name
      localizations(includeCurrent: false, locales: [en, de]) {
     		slug 
      }
    }
    products(locales: $locales,where: $where) {
      description
      id
      slug
      locale
      name
      images(first: 1, where: {}) {
        size
        width
        height
        url(transformation: {image: {resize: {fit: scale, width: 300}}})
      }
    }
 
```


```
  variables: {
    locales: [$ws$dataSource$e51NMZq9hEP__DASH__h4qDUpkCP.search.lang ?? "en"],
      "currentCategoryWhere": {
    "slug": $ws$dataSource$e51NMZq9hEP__DASH__h4qDUpkCP.search.category ?? ""
  },
    where: {
            categories_some: {
              slug: $ws$dataSource$e51NMZq9hEP__DASH__h4qDUpkCP.search.category
            },        
    }
  }
```

**Comment by istarkov** [comment]
# Attach above query to webstudio

**Comment by istarkov** [comment]
Now we can use `Products` resource we have created above

Add collection, name it Products
image

Rename `Collection item` variable on `Product`
image

image

Bind Data to `Products.data.data.products`
image

Bind Item Elements to Product 

image

**Pull request closed without merging**